keepercommander/api.py

```diff
@@ -1596,9 +1649,12 @@ def communicate_rest(params, request, endpoint):
     raise KeeperApiError('Error', endpoint)
 
 
-def communicate(params, request):
+def communicate(params, request, skip_ttk=False):
```

Comment:
What is the point of adding skip_ttk parameter?
It would be nice to keep the existing method signatures unchanged.
This method is one the most used methods in the app.